main.cpp

```diff
@@ -585,14 +490,18 @@ void Synthesizer::handleKeyPress(int keyNumPressed)
 
 void Synthesizer::synthOutput(int style)
 {
-    
     if( style == 1)
     {
         sequenceNotesOn();
         playSound(44,444,333);
         lfo1.changePitch(10.2f,2.2f);
         fxtab1.reverbOutput(10, true);
     }
+    sequenceNotesOn();
+    playSound(33,22,333);
+    lfo1.changePitch(11.2f,5.2f);
+    fxtab1.reverbOutput(18, true);
+    //FIXME what if the style is not 1?
```

Comment:
if style == 1, you do the stuff in the `{ }` and then you do the stuff after the `{ }`
If style != 1, you skip the stuff in the `{ }` and go right to the stuff after the `{ }`

Is this what you want to have happen?  the stuff after the `{ }` is ALWAYS executed, potentially overwriting whatever the stuff in the `{ }` did.  For example, the reverbOutput will be set to 18, and not stay at 10.
- [ ] revise.